test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +44,30 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+    
+    @Test
+    public void isAnyNull() throws Exception{
```

Comment:
As discussed in the tutorial, these test cases can be separated into multiple more specific methods. Thus, you can remove the methods below. Refer to our coding standard: https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html